files/private-chef-cookbooks/private-chef/recipes/bookshelf.rb

```diff
@@ -49,16 +49,16 @@
   to bookshelf_log_dir
 end
 
-template "/opt/opscode/embedded/service/bookshelf/bin/bookshelf" do
-  source "bookshelf.erb"
-  owner OmnibusHelper.new(node).ownership['owner']
-  group OmnibusHelper.new(node).ownership['group']
-  mode "0755"
-  variables(node['private_chef']['bookshelf'].to_hash)
-  notifies :restart, 'runit_service[bookshelf]' if is_data_master?
-end
+## template "/opt/opscode/embedded/service/bookshelf/bin/bookshelf" do
+##   source "bookshelf.erb"
+##   owner OmnibusHelper.new(node).ownership['owner']
+##   group OmnibusHelper.new(node).ownership['group']
+##   mode "0755"
+##   variables(node['private_chef']['bookshelf'].to_hash)
+##   notifies :restart, 'runit_service[bookshelf]' if is_data_master?
+## end
 
-bookshelf_config = File.join(bookshelf_etc_dir, "app.config")
+bookshelf_config = File.join(bookshelf_dir, "sys.config")
```

Comment:
We should note the change in the rendered configuration location, at least internally.